pkg/action/uninstall.go

```diff
@@ -220,15 +219,15 @@ func (u *Uninstall) deleteRelease(rel *release.Release) (kept string, errs []err
 
 	filesToKeep, filesToDelete := filterManifestsToKeep(files)
 	if len(filesToKeep) > 0 {
-		kept = summarizeKeptManifests(filesToKeep, u.cfg.KubeClient, rel.Namespace)
+		kept = summarizeKeptManifests(filesToKeep, u.cfg.KubeClient)
 	}
 
 	for _, file := range filesToDelete {
 		b := bytes.NewBufferString(strings.TrimSpace(file.Content))
 		if b.Len() == 0 {
 			continue
 		}
-		if err := u.cfg.KubeClient.Delete(rel.Namespace, b); err != nil {
+		if err := u.cfg.KubeClient.Delete(b); err != nil {
```

Comment:
is there a way we can refactor `.Wait` to accept a condition? That way we could wait for these resources to reach the deleted state. I've heard people ask about that.

Doesn't have to be done in this PR, just thought if you wondered if that's possible

Reply:
That is my plan!